src/example/App/VariableText.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import createReactClass from 'create-react-class';
 import {shouldComponentUpdate} from 'react/lib/ReactComponentWithPureRenderMixin';
```

Comment:
Just in case... If the only use of `react-height` for you is `react-collapse` - switch to `react-collapse@3`, that one does not have `react-height` dependency anymore.